### ISSUE TITLE
Do Data Visualization of top 50 most Profitable movies and least profitable movies respectively

### ISSUE THREAD

**Issue by Mangalam0512** [DSCKGEC/MovieRecommendationSystem#4]
**Describe the solution I'd like**
Find out the top 50 most profitable movies, and top 50 least profitable movies and do a horizontal Bar Graph.

**Comment by EleonoreBivo** [comment]
Hello Mangalam, I'm a beginner in Python and Git & Guthub, and I'd like to work on this issue as my first real issue. Shall I?

**Comment by Mangalam0512** [comment]
> Hello Mangalam, I'm a beginner in Python and Git & Guthub, and I'd like to work on this issue as my first real issue. Shall I?

Please Register for Ksoc'22 first and then you'll be good to go to work on this issue.
Form link : https://docs.google.com/forms/d/e/1FAIpQLSfpih7bxaVYohhTVxaP7vA-9Ux60ygUx55siI8kO3imoEklTw/viewform

**Comment by RohanHBTU** [comment]
Hello Mangalam, I have already registered for KSOC'22 and I'd like to work on this issue. May I?

**Comment by EleonoreBivo** [comment]
> 

Thank you, Mangalam. Completing the form you sent, I just realized this program is for students. I'm not anymore a student:-) I'll let Rohan work on the issue then and will look for other open source projects for newbies. Best!

**Comment by Mangalam0512** [comment]
Yes Rohan you may start working on this issue. Please state the how much time you require to complete it. Ig 4-5 day will be more than enough.
